applications/tasks/src/tasks_maintenance.erl

```diff
@@ -110,13 +108,6 @@ start(TaskId) ->
         {'error', Reason} -> print_error(Reason)
     end.
 
--spec restart(text()) -> 'no_return'.
-restart(TaskId) ->
```

Comment:
As we discussed multiple times before, the task scheduler was never meant to support restarting tasks. If this was really intended as a debugging command then its implementation would be within 
ifdef(TEST)...endif boundaries.